bucket/toggl.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://www.toggl.com/",
+    "description": "Time Tracking for Tech Companies",
+    "version": "7.5.119",
+    "license": "Proprietary",
```

Comment:
License is BSD-3-Clause apparently

https://github.com/toggl-open-source/toggldesktop/blob/master/LICENSE